blog/models.py

```diff
@@ -0,0 +1,46 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class User(AbstractUser):
+    class Meta:
+        ordering = ("username",)
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        related_name="post"
```

Comment:
It is better to add `related_name` in a plural form of a model name. Please, resolve this issue in all places